app/controllers/users_controller.rb

```diff
@@ -1,12 +1,10 @@
 class UsersController < ApplicationController
-  def index; end
+  before_action :logged_in_user, except: [:new, :create]
+  before_action :correct_user, only: [:edit, :update]
+  before_action :find_user_by_id, only: [:show, :edit, :update, :destroy]
```

Comment:
```suggestion
  before_action :find_user_by_id, only: [:show, :edit, :update, :destroy]
  before_action :correct_user, only: [:edit, :update]
```

Nên tìm hiểu và dùng `%i()` thay vì `[]` nhé

Reply:
updated